src/main/java/gtPlusPlus/xmod/gregtech/recipes/GregtechRecipeAdder.java

```diff
@@ -185,233 +101,49 @@ public boolean addCokeOvenRecipe(ItemStack aInput1, ItemStack aInput2, FluidStac
-        try {
-            try {
-                if ((aFluidOutput == null) || (aInputStack == null)) {
-                    return false;
-                }
-            } catch (final NullPointerException e) {}
-            try {
-                if (aFluidInput == null) {
-                    GTPP_Recipe aRecipe = new GTPP_Recipe(
-                            false,
-                            new ItemStack[] { aInputStack },
-                            new ItemStack[] {},
-                            null,
-                            new int[] {},
-                            new FluidStack[] {},
-                            new FluidStack[] { aFluidOutput },
-                            aDuration,
-                            aEUt,
-                            0);
-                    GTPP_Recipe.GTPP_Recipe_Map.sMatterFab2Recipes.addRecipe(aRecipe);
-                } else {
-                    GTPP_Recipe aRecipe = new GTPP_Recipe(
-                            false,
-                            new ItemStack[] { aInputStack },
-                            new ItemStack[] {},
-                            null,
-                            new int[] {},
-                            new FluidStack[] { aFluidInput },
-                            new FluidStack[] { aFluidOutput },
-                            aDuration,
-                            aEUt,
-                            0);
-                    GTPP_Recipe.GTPP_Recipe_Map.sMatterFab2Recipes.addRecipe(aRecipe);
-                }
-                RECIPEHANDLER_MatterFabricator.debug5(aFluidInput, aFluidOutput, aDuration, aEUt);
-                return true;
-            } catch (final NullPointerException e) {
-                return false;
-            }
-        } catch (final Throwable e) {
-            return false;
-        }
-    }
-
-    @Override
-    public boolean addFuel(final ItemStack aInput1, final ItemStack aOutput1, final int aEU, final int aType) {
-        if (aInput1 == null) {
-            Logger.WARNING("Fuel Input is Invalid.");
+        if (aFluidOutput == null) {
+            Logger.WARNING("Something was null, returning false");
             return false;
         }
-        // new GregtechRecipe(aInput1, aOutput1,
-        // GregTech_API.sRecipeFile.get("fuel_" + aType, aInput1, aEU), aType);
+        GTPP_Recipe aRecipe = new GTPP_Recipe(
+                false,
+                new ItemStack[] {},
+                new ItemStack[] {},
+                null,
+                new int[] {},
+                aFluidInput == null ? new FluidStack[] {} : new FluidStack[] { aFluidInput },
+                new FluidStack[] { aFluidOutput },
+                aDuration,
+                aEUt,
+                0);
+        GTPP_Recipe_Map.sMatterFab2Recipes.addRecipe(aRecipe);
+        RECIPEHANDLER_MatterFabricator.debug5(aFluidInput, aFluidOutput, aDuration, aEUt);
         return true;
```

Comment:
Please, make sure that return of every method is consistent. Right now some methods check change of size of recipe map, another ones just return true.

I think that returning true is much clearer behavior (instead of checking map size). Methods of RecipeAdder of GT also just return true.